sdk/core/azure-core/test/ut/transport_adapter_base.cpp

```diff
@@ -491,82 +425,6 @@ namespace Azure { namespace Core { namespace Test {
     }
   }
 
-  TEST_P(TransportAdapter, SizePutFromFile)
-  {
-    Azure::Core::Url host(AzureSdkHttpbinServer::Put());
-    std::string testDataPath(AZURE_TEST_DATA_PATH);
-
-#if defined(AZ_PLATFORM_POSIX)
-    testDataPath.append("/fileData");
-#elif defined(AZ_PLATFORM_WINDOWS)
-    testDataPath.append("\\fileData");
-#else
-#error "Unknown platform"
-#endif
-
-    Azure::Core::IO::FileBodyStream requestBodyStream(testDataPath);
-    auto request = Azure::Core::Http::Request(
-        Azure::Core::Http::HttpMethod::Put, host, &requestBodyStream, false);
-    {
-      auto response = m_pipeline->Send(request, Azure::Core::Context::ApplicationContext);
-      checkResponseCode(response->GetStatusCode());
-      auto expectedResponseBodySize = std::stoull(response->GetHeaders().at("content-length"));
-
-      CheckBodyFromStream(*response, expectedResponseBodySize);
-    }
-  }
-
-  TEST_P(TransportAdapter, SizePutFromFileDefault)
-  {
-    Azure::Core::Url host(AzureSdkHttpbinServer::Put());
-    std::string testDataPath(AZURE_TEST_DATA_PATH);
-
-#if defined(AZ_PLATFORM_POSIX)
-    testDataPath.append("/fileData");
-#elif defined(AZ_PLATFORM_WINDOWS)
-    testDataPath.append("\\fileData");
-#else
-#error "Unknown platform"
-#endif
-
-    Azure::Core::IO::FileBodyStream requestBodyStream(testDataPath);
-    auto request = Azure::Core::Http::Request(
-        Azure::Core::Http::HttpMethod::Put, host, &requestBodyStream, false);
-    // Make transport adapter to read default chunk size
-    {
-      auto response = m_pipeline->Send(request, Azure::Core::Context::ApplicationContext);
-      checkResponseCode(response->GetStatusCode());
-      auto expectedResponseBodySize = std::stoull(response->GetHeaders().at("content-length"));
-
-      CheckBodyFromStream(*response, expectedResponseBodySize);
-    }
-  }
-
-  TEST_P(TransportAdapter, SizePutFromFileBiggerPage)
```

Comment:
Well, we should not remove the test cases just because the ctor is now private.  We are giving up coverage and other important features.  This is bad

Reply:
I understand, I was just checking if build succeeds first. Fixed this.